kubectl-rook-ceph.sh

```diff
@@ -179,6 +193,50 @@ function run_operator_restart_command () {
   kubectl --namespace "$NAMESPACE" rollout restart deploy/rook-ceph-operator
 }
 
+function path_cm_rook_ceph_operator_config() {
+  if [[ "$#" -ne 2 ]]; then
+    fail_error "require exactly 2 subcommand: $*"
+  fi
+  kubectl --namespace "$NAMESPACE" patch configmaps rook-ceph-operator-config --type json --patch  "[{ op: replace, path: /data/$1, value: $2 }]"
+}
+
+function get_mon_endpoints (){
+  end_of_command_parsing "$@" # end of command tree
+  kubectl --namespace "$NAMESPACE" get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data'| tr -d '"'
+}
+
+function rook_version () {
+  [[ -z "${1:-""}" ]] && fail_error "Missing 'version' subcommand"
+  subcommand="$1"
+  shift # remove the subcommand from the front of the arg list
+  case "$subcommand" in
+    version)
+      run_rook_version "$@"
+      ;;
+    status)
+      run_rook_cr_status "$@"
+      ;;
+    *)
+      fail_error "'operator' subcommand '$subcommand' does not exist"
+      ;;
+  esac
+}
+
+function run_rook_version () {
+  end_of_command_parsing "$@" # end of command tree
+  kubectl --namespace "$NAMESPACE" exec deploy/rook-ceph-operator -- rook version
+  kubectl --namespace "$NAMESPACE" exec deploy/rook-ceph-operator -- ceph versions --conf=/var/lib/rook/rook-ceph/rook-ceph.config
+}
+
+function run_rook_cr_status() {
+  cr_list=$(kubectl --namespace "$NAMESPACE" get crd | awk '{print $1}'| sed '1d')
+  echo "CR status"
+  for cr in $cr_list
+  do
+    echo "$cr": "$(kubectl --namespace "$NAMESPACE" get "$cr" -ojson| jq --monochrome-output '.items[].status')"
```

Comment:
What does this output look like? If there are more than a couple CRs, seems like it will be very verbose. What if we instead had the command be:
```
rook status [CR]
```
Where the default CR is the cephcluster. Then you won't get all output from all CRs together.

Reply:
It was verbose so I went ahead with your suggestions. 

FYI, you can look at the command output in the CI :slightly_smiling_face: